backend/rebutify/urls.py

```diff
@@ -31,6 +31,9 @@
 router.register(r"status/alive", views.StatusViewSet, basename="alive")
 router.register(r"posts", views.PostViewSet, basename="posts")
 router.register(r"arguments", views.ArgumentViewSet, basename="arguments")
+router.register(
+    r"rebuttals/(?P<parentId>\d+)", views.RebuttalViewSet, basename="rebuttals"
```

Comment:
any reason for the `/(?P<parentId>\d+)`?

adding a rebuttal will need a post to just plain `/api/rebuttals`

![image](https://github.com/Monstarrrr/rebutify/assets/107010978/26df9b77-1528-4ee0-87ff-5a293c593440)
This is what it looks like if I remove `/(?P<parentId>\d+)`

Reply:
![image](https://github.com/Monstarrrr/rebutify/assets/107010978/4b1fecdd-5659-47fe-8897-14ecd0a7e4db)
I think I get it...

Reply:
I was thinking of a case where we'd want to GET all rebuttals, but I don't see a use for that. Looks good